grobid-core/src/main/java/org/grobid/core/utilities/GrobidProperties.java

```diff
@@ -569,6 +569,12 @@ public static Integer getPdfToXMLTimeoutMs() {
         return Integer.parseInt(getPropertyValue(GrobidPropertyKeys.PROP_3RD_PARTY_PDFTOXML_TIMEOUT_SEC, "60"), 10) * 1000;
     }
 
+    public static boolean isPdfToXMLValidationEnabled() {
```

Comment:
Not quite sure why we spell `Pdf` and `XML`. Java naming conventions is all caps but personally I prefer `Pdf` and `Xml`.

Reply:
Not quite sure neither :) 
Maybe a compromise between the different convention :D